client/src/components/Admin/AdminLogin.js

```diff
@@ -50,43 +51,58 @@ class AdminLogin extends React.Component {
           </Heading>
           <main>
             <ErrorContainer />
+
             <div className={css.component}>
               <h1>Logg inn som tellekorps</h1>
-              {this.props.meetingExists
-                ? <p>Du må autorisere deg for å få tilgang til denne funksjonaliteten.</p>
-                : <p>Vennligst opprett en generalforsamling</p>}
-              <form action="">
-                {!this.props.meetingExists &&
-                (<div className={css.createGenfors}>
-                  <input
-                    type="text"
-                    placeholder="Tittel"
-                    value={this.state.title}
-                    onChange={e => this.setState({ title: e.target.value })}
-                  />
-                  <input
-                    type="date"
-                    value={formattedDate}
-                    onChange={e => this.setState({ date: new Date(e.target.value) })}
-                  />
-                </div>
-                )}
-                <div className={css.adminPassword}>
-                  <input
-                    type="password"
-                    placeholder="Administratorpassord"
-                    value={this.state.password}
-                    onChange={e => this.setState({ password: e.target.value })}
-                  />
-                </div>
-              </form>
-              <Button
-                background
-                size="lg"
-                onClick={e => this.authenticateAdmin(e)}
-              >
-                Logg inn
-              </Button>
+
+              {(!this.props.loggedIn || this.props.loggedIn && this.props.meetingExists) && (
```

Comment:
I'm a fan of parenthesis if the statement could be unclear, i.e. `!loggedIn || (loggedIn && meetingExists)`